src/rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] It is expected that both the `book` and  `person` instance variables should have Getters and Setters as you did for the `date`. Kindly fix that.

Reply:
@Qoosim Hi, I did set only `attr_reader ` for **book** and for **person** because I have defined setters for both in line 13 and line 18


![image](https://github.com/Diegogagan2587/school-library/assets/61764778/1a75e4f6-84f1-481b-8459-5c318fe2bed9)

Do you think I still need to change it? Thanks for your feedback by the way!

Reply:
After applying the change in my local environment, I'm getting the next from Rubocop:
![image](https://github.com/Diegogagan2587/school-library/assets/61764778/33d94560-c716-4b2e-bf4c-dc1c23f4aa84)
So yes, the setters are defined in lines 13 and 18, That's the reason why I have to include only attribute readers on the top since I have defined special setters that will update the data in both objects, so I will keep the original code in this case, since would cause problems.